easybuild/easyconfigs/s/spglib/spglib-1.9.2_remove_duplicate.patch

```diff
@@ -0,0 +1,13 @@
+debug.h is already in $(spglib_h), which causes make install to fail
+author: Miguel Dias Costa (National University of Singapore)
+--- src/Makefile.in.orig	2016-04-26 16:21:29.900203000 +0800
++++ src/Makefile.in	2016-04-26 16:21:43.531192514 +0800
+@@ -562,7 +562,7 @@
+ version.h
+ 
+ libsymspg_la_SOURCES = $(spglib_c) $(spglib_h)
+-pkginclude_HEADERS = $(spglib_h) debug.h
++pkginclude_HEADERS = $(spglib_h)
```

Comment:
huh, how can this ever have worked... maybe consider reporting this back upstream?